src/test/java/Problem2Test.java

```diff
@@ -8,7 +8,31 @@
     // Okay to change
     @Test
     public void testBubbleSort() {
-        assertTrue(false);
-        // add your tests here
+        int [][] inputs = {
+                {3, 2, 7, 4, 8},
+                {5, 4, 3, 2, 1},
+                {23, 63, 26, 87, 57},
+                {8, 4},
+                {7, 7, 7, 4, 4,},
+
+        };
+        int [][] answer = {
```

Comment:
great test!

consider add a few more for edge cases. for example, empty array, null array, single-element array.